docs/tools/erc725js/providers.md

```diff
@@ -20,14 +20,14 @@ const RPC_URL = 'https://rpc.testnet.lukso.network';
 const erc725 = new ERC725([], '0x...', RPC_URL);
 ```
 
-## Ethereum (injected provider from extension)
+## Injected Extension Provider
```

Comment:
This file needs to be updated in the source repo of erc725js as the files in this folder are copied by the CI in the docs repo.